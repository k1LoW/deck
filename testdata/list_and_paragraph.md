# List and paragraph

---

# Title

- A
- B

<br><br>

**C**
D
E<br>*F*
