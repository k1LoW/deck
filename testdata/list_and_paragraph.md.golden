[
  {
    "layout": "",
    "titles": [
      "List and paragraph"
    ]
  },
  {
    "layout": "",
    "titles": [
      "Title"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "A"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "B"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "\n"
              },
              {
                "value": "\n"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "C",
                "bold": true,
                "softLineBreak": true
              },
              {
                "value": "D",
                "softLineBreak": true
              },
              {
                "value": "E"
              },
              {
                "value": "\n"
              },
              {
                "value": "F",
                "italic": true
              }
            ]
          }
        ]
      }
    ]
  }
]